doc-content/enterprise-only/integration/dev-studio-installing-jbpm-runtime-environment-proc.adoc

```diff
@@ -0,0 +1,24 @@
+[id='dev-studio-setting-jbpm-runtime-environment-proc']
+= Installing JBoss BPM Suite runtime environment
+
+In order to create Red Hat JBoss BPM Suite projects in Red Hat Developer Studio, it is necessary to install the {PRODUCT} runtime environment.
+
+A runtime environment is a collection of JAR files that represent a specific release of the software and includes the required libraries for compiling and running your business asset.
+
+.Prerequisite
+Red Hat Developer Studio is installed.
+
+.Procedure
+. Download the {ENGINE}:
+.. Log in to the https://access.redhat.com[Red Hat Customer Portal].
+.. Click *DOWNLOADS* at the top of the page.
+.. On the *Product Downloads* page that opens, navigate to the JBOSS DEVELOPMENT AND MANAGEMENT section, and click *{PRODUCT}*.
+.. On the *Software Downloads* page, download *{PRODUCT} {PRODUCT_VERSION_LONG} Add-Ons* (`{PRODUCT_FILE}-add-ons.zip`).
+.. Extract `{PRODUCT_FILE}-add-ons.zip` and then extract the Drools runtime environment JAR files located in `{PRODUCT_FILE}-add-ons/{PRODUCT_FILE}-{URL_COMPONENT_ENGINE}.zip`.
```

Comment:
Ok, if you are sure this is correct:
Drools runtime environment

Reply:
Agreed, adopted